src/BottomNavigation/BottomNavigationItem.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import shallowEqual from 'recompose/shallowEqual';
+import EnhancedButton from '../internal/EnhancedButton';
+
+function getStyles(props, context) {
+  const {selected} = props;
+
+  const {muiTheme} = context;
+  const {bottomNavigation} = muiTheme;
+
+  const color = selected ? bottomNavigation.selectedColor :
+                           bottomNavigation.unselectedColor;
+  const styles = {
+    root: {
+      transition: 'all 0.3s',
```

Comment:
Agreed, it should be their responsibility. In that case I'll restrict the transition to `color` and `font-size`.